api/HTMLSourceElement.json

```diff
@@ -48,6 +48,54 @@
           "deprecated": false
         }
       },
+      "height": {
+        "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/embedded-content-other.html#dom-dim-height",
```

Comment:
@sideshowbarker do you think we should use this URL or https://html.spec.whatwg.org/multipage/embedded-content-other.html#attr-dim-height here?

Reply:
> @sideshowbarker do you think we should use this URL or [html.spec.whatwg.org/multipage/embedded-content-other.html#attr-dim-height](https://html.spec.whatwg.org/multipage/embedded-content-other.html#attr-dim-height) here?

I think that because this is for the API, we shouldn’t use the one for content/markup attribute. So that seems to leave us with https://html.spec.whatwg.org/multipage/embedded-content-other.html#dom-dim-height (the existing spec URL in this PR).

In general we have quite a number of these cases where DOM/IDL attributes, the anchor/target text just says, *“This should reflect the content attribute.”* But arguably the fact that the spec doesn’t state any requirements beyond just that is a significant piece of information. In other words, it’s useful for developers to know that this is a case where there are no requirements beyond just reflecting the content attribute.

Reply:
Thanks Mike, that makes sense. I actually missed that the https://html.spec.whatwg.org/multipage/embedded-content-other.html#attr-dim-height doesn't say "IDL attribute" and assumed they were talking about the same thing.